server/app/com/xsn/explorer/data/anorm/dao/TransactionPostgresDAO.scala

```diff
@@ -426,14 +431,15 @@ class TransactionPostgresDAO @Inject() (
     SQL(
       s"""
         |WITH TXS AS (
-        |   SELECT txid, blockhash, time, size
-        |   FROM transactions
+        |   SELECT txid, blockhash, time, size, sent, received
+        |   FROM transactions t
+        |   WHERE $includeZeroTransactions 
+        |     OR sent > 0 
```

Comment:
I haven't run `explain` but its worth checking the query plan for this vs `max(sent, received)`, or even create an index for `max(sent, received)` because we don't need the individual ones.

Reply:
yeah, sounds like just one index instead of two should be better

Reply:
maybe sent + received > 0 will make the intent of the query clearer thant max(sent, received) > 0